### PR TITLE
outgoing log messages

### LINKED ISSUE #79: Add outgoing packets logging to console.
Currently only incoming packets are being logged. We should also log outgoing packets to help with debugging.

### PR DESCRIPTION
**Pull request by IDjinn**
closes #79

### CODE REVIEW

File: HabboHotel/GameClients/GameClient.cs [hunk 1]
```diff
@@ -84,7 +86,13 @@ public void SendWhisper(string message, int colour = 0)
 
     public void SendNotification(string message) => SendPacket(new BroadcastMessageAlertComposer(message));
 
-    public void SendPacket(IServerPacket message) => GetConnection().SendData(message.GetBytes());
+    public void SendPacket(IServerPacket message)
+    {
+        #if DEBUG
```
Comment (80O) [inline HabboHotel/GameClients/GameClient.cs:91]: Preprocessor directive is not needed, as you call Log.Debug() already, which is disabled in release (through nlog).

File: HabboHotel/GameClients/GameClient.cs [hunk 2]
```diff
@@ -13,6 +14,7 @@ namespace Plus.HabboHotel.GameClients;
 
 public class GameClient
 {
+    private static readonly ILogger Log = LogManager.GetLogger("Plus.HabboHotel.GameClients.GameClient");
```
Comment (80O) [inline HabboHotel/GameClients/GameClient.cs:17]: Use `GetCurrentClassLogger()` instead.

**Comment by IDjinn** [comment]
done.

File: HabboHotel/GameClients/GameClient.cs [hunk 3]
```diff
@@ -84,7 +86,11 @@ public void SendWhisper(string message, int colour = 0)
 
     public void SendNotification(string message) => SendPacket(new BroadcastMessageAlertComposer(message));
 
-    public void SendPacket(IServerPacket message) => GetConnection().SendData(message.GetBytes());
+    public void SendPacket(IServerPacket message)
+    {
+        Log.Debug("Sending Packet [{packetId}]({name}) to user [{userId}]", message.Id, message.GetType().Name, GetHabbo()?.Id);
```
Comment (80O) [inline HabboHotel/GameClients/GameClient.cs:91]: Use `_habbo` instead of `GetHabbo()`

**Pull request merged**